content/sensu-go/5.19/reference/backend.md

```diff
@@ -1243,12 +1244,17 @@ event-log-file: "/var/log/sensu/events.log"{{< /highlight >}}
 ### Log rotation
 
 Event logging supports log rotation via the _SIGHUP_ signal.
-First, rename (move) the current log file.
+To manually rotate event logs, first rename (move) the current log file.
 Then, send the _SIGHUP_ signal to the sensu-backend process so it creates a new log file and starts logging to it.
```

Comment:
@cwjohnston seems like this might be the right place to mention the reload issue with clustered backends prior to 5.19.3, but I'm not sure if that only applies to `systemctl reload` or if it would happen for the sysvinit example too.

Reply:
Unfortunately I believe it applies equally to sysvinit and systemd. Sending `SIGHUP` manually, `systemctl reload sensu-backend` and /etc/init.d/sensu-backend reload` are all doing the same thing.